MVVMRxSwiftDemo/Sources/Presentation/Modules/Home/HomeHeader/HomeHeaderView.swift

```diff
@@ -34,12 +34,12 @@ final class HomeHeaderView: UIView {
     }
 
     func bind(_ viewModel: HomeHeaderViewModel) {
```

Comment:
Please use `HomeHeaderViewModelProtocol` instead, so in the View, we can only access to input or output
```suggestion
    func bind(_ viewModel: HomeHeaderViewModelProtocol) {
```